internal/pkg/client/oci/ocisif.go

```diff
@@ -168,10 +195,7 @@ func layoutToOciSif(layoutDir string, digest v1.Hash, imageDest, workDir string)
 		return fmt.Errorf("while replacing layers: %w", err)
 	}
 
-	sylog.Infof("Writing oci-sif image...")
-	if err := lp.ReplaceImage(img, match.Digests(digest)); err != nil {
```

Comment:
Note that this deletion means that the squashed, mutated image is only written into the final SIF, and not back into the temporary layout.

I had left it writing into the temporary layout for some speculative reason that escapes me, previously... but confirmed it is not needed in this PR... which should be the final significant change for pkg/client/oci/ocisif